libraries/state_history/include/eosio/state_history/log.hpp

```diff
@@ -19,14 +18,13 @@
 #include <boost/iostreams/filter/zlib.hpp>
 #include <boost/iostreams/filtering_streambuf.hpp>
 #include <boost/iostreams/restrict.hpp>
+#include <boost/iostreams/operations.hpp>
+#include <boost/iostreams/copy.hpp>
 
 #include <fstream>
```

Comment:
I don't think fstream, boost/asio.hpp, fc/bitutil.hpp, fc/io/cfile.hpp are used.

Reply:
removed those in 8d716d2